tests/integration/operator/main_test.go

```diff
@@ -0,0 +1,30 @@
+//  Copyright Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package operator
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/resource/environment"
+)
+
+func TestMain(m *testing.M) {
+	framework.
+		NewSuite("operator_controller", m).
+		RequireEnvironment(environment.Kube).
+		RequireSingleCluster().
+		Run()
```

Comment:
I think we should run the `istioctl operator init` here and later only `ApplyConfigOrFail` the IOP yaml

Reply:
done